src/crimson/os/seastore/journal/circular_bounded_journal.h

```diff
@@ -30,7 +30,27 @@ using NVMeBlockDevice = nvme_device::NVMeBlockDevice;
 /**
  * CircularBoundedJournal
  *
- * TODO: move record from CircularBoundedJournal to RandomBlockManager
+ * 
+ * CircularBoundedJournal (CBJournal) is the journal that works like circular
+ * queue. With CBJournal, Seastore will append some of the records if the size
+ * of the record is small (most likely metadata), at which point the head
+ * (written_to) will be moved. Then, eventually, Seastore applies the records
+ * in CBjournal to RBM---the tail (applied_to) is advanced (TODO).
+ *
+ * - Commit time
+ * After submit_record is done, last_committed_record_base and written_to are
+ * advanced---last_committed_record_base indicates the last committed record and
```

Comment:
"are advanced" are these fields in the header?  Do you rewrite the header on each commit?

Reply:
Sorry for confusing you. CBJournal will rewrite the fields, but these fields are in memory (without storing the fields permanently).

After submit_record is done, last_committed_record_base and written_to are increased (these are in memory values)

Reply:
Why?  What purpose do they serve?

Reply:
Removed last_committed_record_base and rework is done based on your comments. See the updated commits.